routes/discordactions.js

```diff
@@ -43,21 +45,33 @@ router.patch(
   checkIsVerifiedDiscord,
   updateDiscordImageForVerification
 );
-router.put("/group-idle", authenticate, authorizeRoles([SUPERUSER]), setRoleIdleToIdleUsers);
-router.put("/group-idle-7d", authenticate, authorizeRoles([SUPERUSER]), setRoleIdle7DToIdleUsers);
+router.put(
+  "/group-idle",
+  authorizeAndAuthenticate([ROLES.SUPERUSER], [Services.CRON_JOB_HANDLER]),
+  setRoleIdleToIdleUsers
+);
+router.put(
+  "/group-idle-7d",
+  authorizeAndAuthenticate([ROLES.SUPERUSER], [Services.CRON_JOB_HANDLER]),
+  setRoleIdle7DToIdleUsers
+);
 router.post(
   "/nicknames/sync",
   authenticate,
   authorizeRoles([SUPERUSER]),
   checkIsVerifiedDiscord,
   updateDiscordNicknames
 );
-router.post("/nickname/status", verifyCronJob, validateUpdateUsersNicknameStatusBody, updateUsersNicknameStatus);
+router.post(
+  "/nickname/status",
+  authorizeAndAuthenticate([ROLES.SUPERUSER], [Services.CRON_JOB_HANDLER]),
```

Comment:
This should have been **/nickames/sync** not **/discord-actions/nickname/status**
Fixing it